app/views/customers/_form.html.erb

```diff
@@ -53,6 +53,33 @@
       <%= person_fields.text_field :last_name, id: :person_last_name, class: "form-control" %>
     </div>
   <% end %>
+  
+  <%= form.fields_for :payment, customer.payment do |payment_fields| %>
+    <div class="panel panel-default">
+      <div class="panel-heading  text-center"><h3>Payment Information</h3></div>
+      <div class="panel-body">
+        <div class="form-group">
+          <%= payment_fields.label :card_name %>
+          <%= payment_fields.text_field :card_name, id: :payment_card_name, class: "form-control" %>
+        </div>
+
+        <div class="form-group">
+          <%= payment_fields.label :card_num %>
+          <%= payment_fields.number_field :card_num, id: :payment_card_num, class: "form-control" %>
```

Comment:
```suggestion
          <%= payment_fields.text_field :card_num, id: :payment_card_num, class: "form-control" %>
```

Reply:
Also please change the validations for payment.rb to:
```
  validates :card_num, presence: true, length: { is: 16 }, numericality: { greater_than: 0, only_integer: true, message: "is invalid" }
  validates :cvc, presence: true, length: { is: 3 }, numericality: { greater_than: 0, only_integer: true, message: "is invalid" }
```
or whatever seems appropriate.